### PR TITLE
Fix some miscellaneous bugs and add minor improvements in response to user feedback

### PR DESCRIPTION
**Pull request by jeffjennings**
From emails in the last couple days, users have noted a couple bugs in the code that arise under certain input parameter choices. This PR fixes these and adds more tests for the associated calls. 

There's also a minor expansion a user requested, to fit for only dRA and dDec (providing inc and PA), implemented here.

Can you check especially carefully the changes in geometry.py in ec88bc6?

### CODE REVIEW

File: frank/geometry.py [hunk 1]
```diff
@@ -339,6 +344,9 @@ def fit(self, u, v, V, weights):
             Weights on the visibilities
         """
 
+        if self._inc_pa:
```
Comment (rbooth200) [inline frank/geometry.py:347]: Can we have a check to ensure that we are fitting something? i.e. one of inc_pa and phase_centre is None. Same for FitGeometryFourierBessel

File: frank/parameter_descriptions.json [hunk 2]
```diff
@@ -12,12 +12,14 @@
 
   "modify_data" : {
     "baseline_range"   : "(Deprojected) baseline range outside of which visibilities are truncated, unit=[\\lambda]",
-    "correct_weights"  : "Whether to estimate the data's weights (as they may be unknown for mock data)"
+    "correct_weights"  : "Whether to estimate the data's weights (as they may be unknown for mock data)",
+    "norm_wle"         : "Wavelength (unit=[m]) by which to normalize the (u, v) points. Not needed if the (u, v) points are already in units of [\\lambda]"
```
Comment (rbooth200) [inline frank/parameter_descriptions.json:16]: Can you be explicit that the wavelength is used to convert the uv points to radians? "Normalize" is a bit confusing

File: frank/tests.py [hunk 3]
```diff
@@ -346,6 +347,10 @@ def test_pipeline_no_phase():
     """Check the full fit pipeline when fitting for the disc's inc, PA"""
     _run_pipeline('gaussian', False)
 
+def test_pipeline_no_inc_no_pa():
```
Comment (rbooth200) [inline frank/tests.py:350]: I can't see where fit_inc_pa is used in `_run_pipeline`

File: frank/geometry.py (hunk 1 above)
Comment (jeffjennings) [reply frank/geometry.py]: Good point, fixed in 1eae50b

File: frank/parameter_descriptions.json (hunk 2 above)
Comment (jeffjennings) [reply frank/parameter_descriptions.json]: yeah, fixed in d489249

File: frank/tests.py (hunk 3 above)
Comment (jeffjennings) [reply frank/tests.py:351]: good catch! I left out the line. Added in 79a2cab

**Comment by jeffjennings** [comment]
I think this is ready to merge now unless you have other requested changes.

**Pull request merged**